src/components/site/Practice.jsx

```diff
@@ -19,79 +22,111 @@ class Practice extends React.PureComponent {
         ...prevState,
         data: res.data,
       }));
-      this.parseDataEcharts();
     });
   }
 
-  parseDataEcharts() {
-    const dataArr = [];
-    const { data } = this.state;
-    if (Object.keys(data).length !== 0) {
-      for (let i = 0; i < data.galaxies.length; i += 1) {
-        const point = data.galaxies[i];
-        const coor = [];
-        coor.push(point.RA);
-        coor.push(point.Dec);
-        dataArr.push(coor);
-      }
-      this.setState(prevState => ({
-        ...prevState,
-        graphData: dataArr,
-      }));
-    }
-  }
-
-  onChartClick(params){
-    console.log(params);
-    console.log(params.color)
+  onChartClick(params) {
+    this.setState(prevState => ({
+      ...prevState,
+      clickedPt: params.value,
+    }));
   }
 
   getEvent() {
     const onEvents = {
-        'click': this.onChartClick,
-    }
+      click: this.onChartClick,
```

Comment:
I tried to dig into the docs and got a lil overwhelmed, so I'm not sure how easy it would be to do this, but I'd like to see a click on NOT a data point result in deselecting anything selected.  Know what I mean?  Like if there's a data point selected, and then I click on an empty part of the graph it should result in unselecting any selected points.  If this looks at all like an overly complicated or WTF thing to get into, don't, instead create a new JIRA ticket for it and we'll get to it later on.

Reply:
This is a little more involved. I want to work more on the click function anyways, so I will add this into it.